includes/class-wppo-cli-command.php

```
@@ -0,0 +1,373 @@
+				/* translators: %s: Database action name */
+				WP_CLI::error( sprintf( __( 'Invalid database action "%s". Did you mean "wp wppo database cleanup"?', 'performance-optimisation' ), $action ) );
+				return;
+			}
+
+			$type = $assoc_args['type'] ?? 'all';
+
+			require_once WPPO_PLUGIN_PATH . 'includes/class-log.php';
+			require_once WPPO_PLUGIN_PATH . 'includes/class-database-cleanup.php';
+
+			if ( 'all' === $type ) {
+				$results = Database_Cleanup::clean_all();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Database_Cleanup' in method 'database'. (undefined)

(StaticAccess)